src/index.ts

```diff
@@ -11,6 +11,7 @@ import Divisor from './components/Divisor';
 import Dropdown, { DropdownV1 } from './components/Dropdown';
 import ErrorBox from './components/ErrorBox/ErrorBox';
 import Icon from './components/Icon/Icon';
+import IconTooltip from './components/IconTooltip/IconTooltip';
```

Comment:
rename IconToolTip.tsx as index.tsx (preferred going forward - we have to clean up other files I agree).. with renamed file, this will be
```
import IconTooltip from './components/IconTooltip/';
```